.github/workflows/release.yaml

```diff
@@ -260,10 +260,20 @@ jobs:
         build_binaries_macos,
         release_docker_image,
       ]
-    runs-on: ubuntu-latest
+    runs-on: macos-12
```

Comment:
Does the entire intellij release step run on macos now? Was it tested?

Reply:
wait.. no need to do it. run it on ubuntu, just wget the version like the other job does.

Reply:
oh right, just saw the vscode fix was already done